### ISSUE TITLE
Upgrade from v0.1.2 to v0.2.1 throws an error

### ISSUE THREAD

**Issue by DamianGlowala** [vite-pwa/nuxt#89]
Tried a fresh install, not working either. Happy to provide any further information.

![image](https://github.com/vite-pwa/nuxt/assets/48835293/badf5190-0dc5-4603-973f-f448977878a4)

```
------------------------------
- Operating System: Windows_NT
- Node Version:     v18.13.0
- Nuxt Version:     3.8.1
- CLI Version:      3.9.1
- Nitro Version:    2.7.2
- Package Manager:  pnpm@8.10.3
------------------------------
```

**Comment by userquin** [comment]
weird, have you run `nuxt prepare` script?

![imagen](https://github.com/vite-pwa/nuxt/assets/6311119/08660bf3-b445-4883-acac-78292c5593ec)

**Comment by userquin** [comment]
ok, found the problem, fixing it...

**Comment by userquin** [comment]
@DamianGlowala can you show me your pwa configuration? I guess are using `manifest: false`... can you confirm?

**Comment by DamianGlowala** [comment]
This is what I currently have set up:
```
pwa: {
    client: {
        installPrompt: true
    }
}
```

**Comment by userquin** [comment]
I'll release a new version, check this comment https://github.com/vite-pwa/nuxt/issues/84#issuecomment-1802641770

**Comment by userquin** [comment]
@DamianGlowala should be fixed in `v0.2.2`, test it without `manifest` and then add empty `manifest` (later you should add manifest entries with your data and icons).
